### ISSUE TITLE
Add UI-support for logging

### ISSUE THREAD

**Issue by ledex** [spaceuptech/mission-control#28]
When [space-cloud/pull/508](https://github.com/spaceuptech/space-cloud/pull/508) is merged, UI-support for configuring the logging should be added.

**Comment by YourTechBud** [comment]
@Jayesh333 can you come up with a design for this. Lets make this live in `v0.14.0`

**Comment by HeyItsJs** [comment]
Is this configurable via UI? Afaik this is configurable via flags only right? @YourTechBud

**Comment by YourTechBud** [comment]
That's right. We should open up a nee issue in space cloud regarding thid
